doc/reference/tooling/tt_cli/cartridge.rst

```diff
@@ -0,0 +1,514 @@
+.. _tt_cartridge:
+
+Managing a Cartridge application
+================================
+
+..  code-block:: bash
+
+    tt cartridge COMMAND [command options]
+
+``tt cartridge`` manages a Cartridge application.
+``COMMAND`` is one of the following:
+
+*   :ref:`admin <tt_cartridge_admin>`
+*   :ref:`bench <tt_cartridge_bench>`
+*   :ref:`failover <tt_cartridge_failover>`
+*   :ref:`repair <tt_cartridge_repair>`
+*   :ref:`replicasets <tt_cartridge_replicasets>`
+
+
+.. _tt_cartridge_admin:
+
+admin
+-----
+
+..  code-block:: bash
+
+    tt cartridge admin ADMIN_FUNC_NAME [options]
+
+``tt cartridge admin`` calls admin functions provided by the application.
```

Comment:
Maybe add a link to the admin extension (as in the Cartridge doc)?
No explanations, just a link under "admin functions".